encoding/m3tsz/encoder.go

```diff
@@ -140,7 +157,7 @@ func (enc *encoder) writeAnnotation(ant ts.Annotation) {
 		return
 	}
 	scheme := enc.opts.GetMarkerEncodingScheme()
-	writeSpecialMarker(enc.os, scheme, scheme.Annotation())
+	encoding.WriteSpecialMarker(enc.os, scheme, scheme.Annotation())
```

Comment:
Does `WriteSpecialMarker ` need to be in `encoding` now that tsz is combined with m3ts?